examples/gke/deploy-paligemma-2-with-tgi/README.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Deploy PaliGemma2 with TGI DLC on GKE
+type: inference
+---
+
+# Deploy PaliGemma2 with TGI DLC on GKE
+
+PaliGemma 2 is the latest multilingual vision-language model released by Google, that combines the Gemma2 language model with the SigLIP vision model, enabling it to process both images and text inputs to generate text outputs for various tasks including captioning, visual question answering, and object detection. Text Generation Inference (TGI) is a toolkit developed by Hugging Face for deploying and serving LLMs, with high performance text generation. Google Kubernetes Engine (GKE) is a fully-managed Kubernetes service in Google Cloud that can be used to deploy and operate containerized applications at scale using Google Cloud infrastructure.
+
+This example showcases how to deploy Google PaliGemma2 from the Hugging Face Hub on a GKE Cluster, running a purpose-built container to deploy LLMs and VLMs in a secure and managed environment with the Hugging Face DLC for TGI. Additionally, this example also presents different scenarios or use-cases where PaliGemma2 can be used.
```

Comment:
```suggestion
This example showcases how to deploy Google PaliGemma 2 from the Hugging Face Hub on a GKE Cluster, running a purpose-built container to deploy LLMs and VLMs in a secure and managed environment with the Hugging Face DLC for TGI. Additionally, this example also presents different scenarios or use-cases where PaliGemma2 can be used.
```
PaliGemma 2* as Google mentioned we should put like that

Reply:
Thanks for the heads up, will check if there's any other `PaliGemma2` instead of `PaliGemma 2` around the example 👍🏻